src/main/java/ch/njol/skript/Skript.java

```diff
@@ -285,6 +285,18 @@ private static boolean checkServerPlatform() {
 		// If nothing got triggered, everything is probably ok
 		return true;
 	}
+
+	private static final List<Class<? extends Hook<?>>> disabledHookRegistrations = new ArrayList<>();
```

Comment:
I think a Set would be a better collection here since this list doesn't need to contain dupes and order does not matter